core-services/prow/02_config/_plugins.yaml

```diff
@@ -1974,6 +1974,28 @@ repo_milestone:
     maintainers_friendly_name: odo maintainers
     maintainers_id: 3240507
     maintainers_team: odo-maintainers
+require_matching_label:
```

Comment:
I don't think I can accept it in this place. It should be added to _pluginconfig in repo conf.

Reply:
Ups, this plugin is not supported using shared configuration: https://github.com/kubernetes/test-infra/blob/master/prow/plugins/config.go#L1871~#L1876
In any way I would not like to add to the main prow for every repo, specific info about one org. So, the most likely upstream feature is needed to support this.